test/langtools/tools/doclint/html/HtmlVersionTagsAttrsTest.java

```diff
@@ -151,7 +151,7 @@ public void notSupportedTags_html5() { }
      * <div class="desc">Desc</div>
      * </template>
      * <p>Test current time is <time>10:00</time> at night</p>
-     * <p>Test <wbr>WBR</wbr> text</p>
+     * <p>Test<wbr>text</p>
```

Comment:
this is a positive test shouldnt we have negative tests too?

Reply:
I guess I felt this was close to a `noreg-trivial` change to an element that is rarely used, at least in doc comments. The only reason there's a test at at is because it is a holdover from the HTML5 conversion.

Reply:
got it, sounds good